runelite-client/src/main/java/net/runelite/client/plugins/statusbars/StatusBarsOverlay.java

```diff
@@ -190,6 +191,13 @@ else if (poisonState >= 1000000)
 
 				for (final StatChange c : statsChanges.getStatChanges())
 				{
+					Integer string = Ints.tryParse(c.getTheoretical().substring(1));
+					if (string == null)
```

Comment:
this is wrong, because if the number starts with - or nothing it just deletes that char. Also, in case this matches you do nothing with `string` var